include/ck_tile/ops/fmha/kernel/fmha_fwd_kernel.hpp

```diff
@@ -344,6 +346,14 @@ struct FmhaFwdKernel
         }
         if constexpr(kHasDropout)
         {
+            if (drop_seed_ptr != nullptr)
+            {   
+                kargs.drop_seed = *(reinterpret_cast<const uint64_t*>(drop_seed_ptr));
```

Comment:
Are both the `drop_seed_ptr` & `drop_offset_ptr` point to host memory?

Reply:
you should move this into operator(). Because drop_seed_ptr point to device memory

Reply:
Stale comment, no longer valid after the latest code changes.